### PR TITLE
Refactor/replace dynamic locales

### PR DESCRIPTION
**Pull request by queuing4oranges**
-submodule update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,20 @@
 
 ## Release Candidate
 
+### Dependencies
```
Comment (petrKavulok) [inline CHANGELOG.md]: remove `Dependencies`. Submodule updates should go under `Features` only

**Review by petrKavulok — CHANGES_REQUESTED** [review]
minor changelog.md update required. Also, please, resolve conflicts before merging. Otherwise good job! :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,14 @@
 
 ### Features
 
-- ASAB WebUI submodule version bump [[4675693](https://github.com/TeskaLabs/asab-webui/commit/46756931bf295b40d00cf23ab8849233c35fed24)] commit (INDIGO Sprint 230623, [!46](https://github.com/TeskaLabs/seacat-admin-webui/pull/46))
+- ASAB WebUI submodule version bump [[309a833](https://github.com/TeskaLabs/asab-webui/commit/06af558428a2106e14d00d3fdb0b9457b761f787)] commit (INDIGO Sprint 230623, [!49](https://github.com/TeskaLabs/seacat-admin-webui/pull/49))
```
Comment (petrKavulok) [inline CHANGELOG.md]: mismatch between the commit hash in URI and the short version of commit hash in square brackets

**Review by petrKavulok — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,12 +4,14 @@
...
 
 ### Refactoring
 
 - Add styles for primary dropdown-toggle (INDIGO Sprint 230623, [!428](https://github.com/TeskaLabs/asab-webui/pull/428))
 
+- Submodule update - refactored dynamic locales (INDIGO Sprint 230623, [!432](https://github.com/TeskaLabs/asab-webui/pull/432))
```
Comment (Pe5h4) [inline CHANGELOG.md]: @queuing4oranges You dont need to write here `Submodule update`.

`Refactored dynamic locales` is just fine

**Review by Pe5h4 — APPROVED** [review]

**Review by petrKavulok — APPROVED** [review]
good job 🥇

**Pull request merged**